lib/Common/Memory/RecyclerTelemetryInfo.cpp

```diff
@@ -35,11 +36,14 @@ namespace Memory
 
     RecyclerTelemetryInfo::~RecyclerTelemetryInfo()
     {
-        if (this->hostInterface != nullptr && this->passCount > 0)
+        if (this->hostInterface != nullptr)
         {
             AssertOnValidThread(this, RecyclerTelemetryInfo::~RecyclerTelemetryInfo);
-            this->hostInterface->TransmitTelemetry(*this);
-            this->FreeGCPassStats();
+            if (this->gcPassStats.Empty() == false)
```

Comment:
I'm curious, why do you prefer `this->gcPassStats.Empty() == false` over `!this->gcPassStats.Empty()` ?

Reply:
Used to work with a guy who'd complain in every CR that it was easier for him to read `x == false` vs `!false`, since the `!` is more easily missed.  :) As I was going through this code, I missed `!`, and went back & changed stuff up to try to be consistent & explicit. 

That said, I don't have a strong preference here & happy to change.